### PR TITLE
feat: Toggle automation status

### PR DESCRIPTION
**Pull request by muhsin-k**
## Description
Allows users to activate/deactivate a automation

https://user-images.githubusercontent.com/15716057/154130570-054f4bd7-a74b-4728-95dc-37e649aa95dc.mp4

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue [hunk 1]
```diff
@@ -20,7 +20,8 @@
         data-view-component="true"
         label="Beta"
         class="beta"
-        >Beta
+      >
+        Beta
```
Comment (nithindavid) [inline app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue:24]: i18n here

File: app/javascript/shared/assets/stylesheets/spacing.scss [hunk 2]
```diff
@@ -8,6 +8,7 @@
   --space-one: 1rem;
   --space-slab: 1.2rem;
   --space-normal: 1.6rem;
+  --space-one-point-five: 1.5rem;
```
Comment (nithindavid) [inline app/javascript/shared/assets/stylesheets/spacing.scss:11]: We don't usually need this. Would be better to use as a local variable.

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/shared/assets/stylesheets/spacing.scss (hunk 2 above)
Comment (fayazara) [reply app/javascript/shared/assets/stylesheets/spacing.scss]: Resolved

File: app/javascript/dashboard/routes/dashboard/settings/automation/Index.vue [hunk 3]
```diff
@@ -246,4 +290,41 @@ export default {
...
   margin: 0;
 }
+.toggle-button {
+  background-color: var(--s-200);
+  position: relative;
+  display: inline-flex;
+  height: 19px;
+  width: 34px;
+  border: 2px solid transparent;
+  border-radius: 9999px;
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/automation/Index.vue:300]:
```suggestion
  border-radius: var(--border-radius-large);
```

**Comment by iamsivin** [comment]
Tested it in local, the toggle button is working as expected.

**Review by iamsivin — APPROVED** [review]
LGTM

**Comment by fayazara** [comment]
@muhsin-k @nithindavid  Need a review on this.

**Comment by muhsin-k** [comment]
@fayazara The front end code looks good to me

**Pull request merged**